lib/human_player.rb

```diff
@@ -13,7 +13,7 @@ def get_opponent
     Mark::O if @mark == Mark::X 
   end
 
-  def get_next_move(board)
-    @display.ask_player_for_move(board)
+  def get_next_move
+    @display.ask_player_for_move(@mark)
```

Comment:
I think you can refer to the member variables without the '@' sign in the class they are defined (except in the constructor)